src/python/espressomd/lb.pyx

```diff
@@ -542,3 +548,144 @@ cdef class LBFluidRoutines:
 
         def __set__(self, value):
             raise NotImplementedError
+
+cdef class LBSlice:
+    cdef Vector3i node 
+    cdef np.ndarray x_indices
+    cdef np.ndarray y_indices
+    cdef np.ndarray z_indices
+
+    # cdef Vector3i shape
+
+    def __init__(self, key):
+        shape = lb_lbfluid_get_shape()
+        self.x_indices, self.y_indices, self.z_indices = self.get_indices(
+            key, shape[0], shape[1], shape[2])
+
+    def get_indices(self, key, shape_x, shape_y, shape_z):
+        _x_indices = np.atleast_1d(np.arange(shape_x)[key[0]])
+        _y_indices = np.atleast_1d(np.arange(shape_y)[key[1]])
+        _z_indices = np.atleast_1d(np.arange(shape_z)[key[2]])
+        return _x_indices, _y_indices, _z_indices
+
+    def get_values(self, _x_indices, _y_indices,
+                   _z_indices, prop_name, shape_res):
+        res = np.zeros(
+            (_x_indices.size,
+             _y_indices.size,
+             _z_indices.size,
+             *shape_res))
+        for i, x in enumerate(_x_indices):
+            for j, y in enumerate(_y_indices):
+                for k, z in enumerate(_z_indices):
+                    res[i, j, k] = getattr(LBFluidRoutines(
+                        np.array([x, y, z])), prop_name)
```

Comment:
this is repeated in every property and can probably be reused if put into a function